mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/loanaccountdisbursement/LoanAccountDisbursement.java

```diff
@@ -272,5 +224,20 @@ public void onDestroyView() {
 
     }
 
+    @Override
+    public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+        switch (parent.getId()) {
+            case R.id.sp_loan_payment_type:
+                paymentTypeId = mLoanDisburseTemplate.getPaymentTypeOptions().get(position).getId();
+                break;
+
```

Comment:
@ishan1604 can you tell me better way so that parentTypeId assigns to corresponding spinner item.